blog/report.md

```diff
@@ -0,0 +1,71 @@
+# Benchmark Report
+
+## Goal
+We want to give the user the opportunity to know the minimum size of OCP cluster that can stand for his daily logs written and read from Loki. Our goal is to create a table that specifies for each "T-shirt" size the average and max daily read and write capabilities of Loki.
+
+## Environment
+We ran that on OpenShift running on AWS clusters with configuration:
+
+	apiVersion: v1
+	baseDomain: devcluster.openshift.com
+	compute:
+	- architecture: amd64
+	hyperthreading: Enabled
+	name: worker
+	platform:
+		aws:
+		type: m4.16xlarge
+		rootVolume:
+			type: io1
+			size: 500
+			iops: 8000
+	replicas: 6
+	controlPlane:
+	architecture: amd64
+	hyperthreading: Enabled
+	name: master
+	platform: {}
+	replicas: 3
+
+Running with 64 vCPUs and 256 Memory (GiB)
+
+
+## The Proccess Of Creating The Table
+The benchmark tool helps us observe the data (such as CPU, Memory, etc...) of Loki while sending a different amount of logs each test, the logs are sent by a logger that is deployed when the benchmark test is started (the implementation can be seen at https://github.com/ViaQ/cluster-logging-load-client).
```

Comment:
s/helps us observe the data/records operational metrics
s/each test,/in each test.